tests/Vendor/Order/OrderServiceTest.php

```diff
@@ -278,6 +278,153 @@ public function testGenerateMondialRelayLabel(): void
         $this->assertInstanceOf(MondialRelayLabel::class, $result);
     }
 
+    public function testGetOrderAmountsTaxesDetails(): void
+    {
+        $orderService = $this->buildVendorOrderService();
+
+        $order = $orderService->getOrderById(9);
+
+        static::assertInstanceOf(AmountTaxesDetail::class, $order->getTotalsTaxesDetail());
+        static::assertInstanceOf(AmountTaxesDetail::class, $order->getShippingCostsTaxesDetail());
+        static::assertInstanceOf(AmountTaxesDetail::class, $order->getCommissionsTaxesDetail());
+        static::assertInstanceOf(AmountTaxesDetail::class, $order->getVendorShareTaxesDetail());
+
+        static::assertEquals('53.0852', $order->getTotalsTaxesDetail()->getExcludingTaxes());
+        static::assertEquals('1.1148', $order->getTotalsTaxesDetail()->getTaxes());
+        static::assertEquals('54.2', $order->getTotalsTaxesDetail()->getIncludingTaxes());
+
+        static::assertEquals('0', $order->getShippingCostsTaxesDetail()->getExcludingTaxes());
+        static::assertEquals('0', $order->getShippingCostsTaxesDetail()->getTaxes());
+        static::assertEquals('0', $order->getShippingCostsTaxesDetail()->getIncludingTaxes());
+
+        static::assertEquals('0.56', $order->getCommissionsTaxesDetail()->getExcludingTaxes());
+        static::assertEquals('0.112', $order->getCommissionsTaxesDetail()->getTaxes());
+        static::assertEquals('0.672', $order->getCommissionsTaxesDetail()->getIncludingTaxes());
+
+        static::assertEquals('52.5252', $order->getVendorShareTaxesDetail()->getExcludingTaxes());
+        static::assertEquals('1.0028', $order->getVendorShareTaxesDetail()->getTaxes());
+        static::assertEquals('53.528', $order->getVendorShareTaxesDetail()->getIncludingTaxes());
+    }
+
+    public function testGetOrdersAmountsTaxesDetails(): void
+    {
+        $orders = $this->buildVendorOrderService()->listOrders(OrderStatus::COMPLETED());
+
+        $this->assertContainsOnly(OrderSummary::class, $orders);
+        $this->assertCount(4, $orders);
```

Comment:
Rendre le test plus souple dans le cas de l'évolution des fixtures, à noter que l'ordre aussi peut changer
```suggestion
        $this->assertTrue(count($orders) >= 4);
```